xls/dslx/lsp/find_definition.h

```diff
@@ -41,7 +41,8 @@ namespace xls::dslx {
 // colon-reference to a construct in another module will return nullopt.
 std::optional<Span> FindDefinition(const Module& m, const Pos& selected,
                                    const TypeInfo& type_info,
-                                   ImportData& import_data);
+                                   ImportData& import_data,
+                                   const NameDef** name_def_out = nullptr);
```

Comment:
Rather than returning name_def_out through an argument, it seems easier to have this be `std::optional<NameDef*> FindDefinition(...)` and have existing usages update to get the span from the `NameDef`.

What do you think? Is it really annoying to update the usages?

Reply:
Good idea, done, PTAL.